index.html

```diff
@@ -41,6 +40,151 @@ <h1>I'm Tash <br> Glad to see you!</h1>
 			    </div>
 			</div>
 		</section>
+
+		<section class="works">
+			<div class="works1">
+			<img class="snap" src="images/Snap3.png" alt="">
+			<p class="tonic">Tonic</p>
+
+			<div class="grid-container">
+				  <div class="grid-item1"><p>CANOPY</p></div>
+				  <div class="grid-item2"><p>.</p></div>
+				  <div class="grid-item3"><p>Back End Dev</p></div>
+				  <div class="grid-item4"><p>.</p></div>
+				  <div class="grid-item5"><p>2015</p></div>
+			</div>
+			<p class="day">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+			<div class="langs">
+				<button class="btn1" type="submit">html</button>
+				<button class="btn2" type="submit">css</button>
+				<button class="btn3" type="submit">javaScript</button>
+			</div>
+			<button class="btn4" type="submit">See Project</button>
```

Comment:
- [ ] Great job implementing the buttons, but kindly ensure that your buttons have the same selector styling as provided in the Figma design. Right now there's no hover or pressed effect visible at all. You should follow the interactors styling as specified in the Figma design which you can see below
![image](https://user-images.githubusercontent.com/90556221/185597037-1321f327-9daf-4604-b31e-21c7536e39e9.png)